src/coreclr/src/ToolBox/superpmi/superpmi-shared/methodcontext.cpp

```diff
@@ -1289,7 +1289,7 @@ void MethodContext::recTryResolveToken(CORINFO_RESOLVED_TOKEN* pResolvedToken, b
 
     TryResolveTokenValue value;
 
-    value.tokenOut = SpmiRecordsHelper::StoreAgnostic_CORINFO_RESOLVED_TOKENout(pResolvedToken, ResolveToken);
+    value.tokenOut = SpmiRecordsHelper::StoreAgnostic_CORINFO_RESOLVED_TOKENout(pResolvedToken, TryResolveToken); // copypaste artifacts?
```

Comment:
Be advised, although I'm not 100% sure, same below.

Reply:
Yeah, it looks like the wrong map is getting used here. Suspect the keys are conformable and don't collide so we never noticed.

cc @sandreenko

Reply:
Agree, thanks for catching this.